src/main/java/com/nokia/uwr/service/parser/BTSParserImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.nokia.uwr.service.parser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nokia.uwr.scenario.bts.BTSDescription;
+import lombok.AllArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Main implementation
+ *
+ * @author Piotr Stoklosa
+ */
+@Service
+@AllArgsConstructor
+public class BTSParserImpl implements BTSParser {
+
+    private static final Logger LOGGER = LogManager.getLogger(BTSParserImpl.class);
+
+    ObjectMapper mapper;
+
+    /**
+     * Main implementation
+     *
+     * @author Piotr Stoklosa
+     */
+    @Override
+    public String parseBTSDescriptions(List<BTSDescription> descriptions) throws IllegalArgumentException {
+
+        if (descriptions == null) throw new IllegalArgumentException("descriptions is null");
+
+        String json;
+        LOGGER.info("Creating JSON from BTS descriptions...");
```

Comment:
Isn't it a good idea to put there "Creating JSON from BTS descriptions..." + descriptions to show which list we want to parse?